man/ocitools-validate.1.md

```diff
@@ -18,8 +18,15 @@ Validate an OCI bundle
 **--path=PATH
   Path to bundle
 
-**--hooks**
-  Check specified hooks exist and are executable on the host.
+**--host-specific**
+  Check host specific configs.
+  By default, validation only tests for compatibility with a hypothetical host. 
+  With this flag, validation will also run more specific tests to see whether 
+  the current host is capable of launching a container from the configuration. 
+  For example, validating a compliant Windows configuration on a Linux machine 
+  will pass without this flag ("there may be a Windows host capable of 
+  launching this container"), but will fail with it ("this host is not capable 
+  of launching this container").
```

Comment:
nit: there's trailing whitespace here (in case we care in this repository; I don't if we do).